consensus/consortium/v1/consortium.go

```diff
@@ -756,6 +774,35 @@ func (c *Consortium) doCalcDifficulty(signer common.Address, number uint64, vali
 	return new(big.Int).Set(diffNoTurn)
 }
 
+// getValidatorsFromGenesis gets the list of validators from the genesis block support backward compatibility in v1, only used with Snap Sync.
+func (c *Consortium) getValidatorsFromGenesis() ([]common.Address, error) {
+	if c.chainConfig.ConsortiumV2Block == nil {
```

Comment:
What is the purpose of this check? ConsortiumV2Block is the hardfork block of version 2.

Reply:
hm let me remove, this check is valid, maybe mis typo